scripts/aws/deploy.py

```diff
@@ -134,6 +134,8 @@ def update_stage(client, rest_api_id, stage_name, stage_settings):
                         help="Location of analyitcs-api IDA for request routing")
     parser.add_argument('--registrar-host', required=True,
                         help="Location of registrar IDA for request routing")
+    parser.add_argument('--enterprise-catalog-host', required=True,
```

Comment:
The variables for this were added in:
https://github.com/edx/edx-gomatic/pull/718
https://github.com/edx/gomatic-internal/pull/282/